lib/vyasa/sangh.ex

```diff
@@ -34,6 +34,7 @@ defmodule Vyasa.Sangh do
 
   @doc """
   Deletes a particular mark within a particular sheaf and adjusts the ranks of the remaining marks to ensure they are in order.
+  NOTE: @ks0m1c QQ: needs some version of authorisation here, in a multi-user sangh, one user shouldn't be able to delete others' marks and sheafs willy-nilly (unless they are admins).
```

Comment:
Will leave it to the implementation for this function to handle that condition

Reply:
this permissioning model has to be backed by a user model, anon users might have to forced into a user model as a prereq

Reply:
yeah, maybe anon could be part of the lowest-wrung ring of perms that the user model 
handles